jina/drivers/cache.py

```diff
@@ -55,11 +50,21 @@ def on_miss(self, req_doc: 'Document', value: str) -> None:
     def on_hit(self, req_doc: 'Document', hit_result: Any) -> None:
         """Call when cache is hit for a document.
 
-        :param req_doc: the document in the request and hitted in the cache
+        :param req_doc: the document in the request and hit in the cache
         :param hit_result: the hit result returned by the cache
         """
         pass
 
+    @staticmethod
+    def hash(doc: 'Document', fields: List[str]) -> str:
```

Comment:
buildin method `hash` should not be shadowed.

Reply:
Done